scripts/generate-cipherinfo.py

```diff
@@ -0,0 +1,337 @@
+#!/usr/bin/env python
```

Comment:
The script at

    https://gist.github.com/mkrautz/8de1d604d01ea45feba2

should be more maintainable, since it only uses the IETF TLS parameters to generate the output of the table, instead of relying on a combination of 'openssl ciphers' and the IETF SSL parameters file, along with heuristics to Do The Right Thing.